File: planning_meal/src/main/java/in/stackroute/planning_meal/model/recipesToFrontEnd/MealPlanEach.java
package in.stackroute.planning_meal.model.recipesToFrontEnd;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class MealPlanEach {
    private String name;
    private double calories;
    private int servings;
    private String image;
    private String sourceUrl;
}

File: product-webapp-service/webapp/src/app/model/mealPlanEach.ts
export class MealPlanEach {
  name: string;
  calories: number;
  servings: number;
  image: string;
  sourceUrl:string;

  constructor(name: string="", calories: number=0, servings: number=0, image: string="",sourceUrl:string="") {
    this.name = name;
    this.calories = calories;
    this.servings = servings;
    this.image = image;
    this.sourceUrl=sourceUrl;
  }
}